modules/codepipeline/templates/buildspec_scan.yml

```diff
@@ -16,4 +16,4 @@ phases:
       - export PATH=$PATH:./sonar-scanner-4.7.0.2747-linux/bin/
   build:
     commands:
-      - sonar-scanner -Dsonar.projectKey=sample-lambda -Dsonar.sources=. -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=${SONAR_ORG} -Dsonar.host.url=${SONARQUBE_ENDPOINT}
\ No newline at end of file
+      - sonar-scanner -Dsonar.projectKey=revature-sample-lambda -Dsonar.sources=. -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=${SONAR_ORG} -Dsonar.host.url=${SONARQUBE_ENDPOINT}
```

Comment:
The `sample-lambda` project key was already taken when I created my sonarcloud project. Thus I had to choose a different project key and update the buildspec accordingly.